tests/test_sesv2/test_sesv2.py

```diff
@@ -0,0 +1,83 @@
+import boto3
+from botocore.exceptions import ClientError
+import pytest
+from moto import mock_sesv2, mock_ses
+from ..test_ses.test_ses_boto3 import get_raw_email
+
+
+@pytest.fixture(scope="function")
+def ses_v1():
+    """Use this for API calls which exist in v1 but not in v2"""
+    with mock_ses():
+        yield boto3.client("ses", region_name="us-east-1")
+
+
+@mock_sesv2
+def test_send_email(ses_v1):  # pylint: disable=redefined-outer-name
+    # Setup
+    conn = boto3.client("sesv2", region_name="us-east-1")
+    kwargs = dict(
+        FromEmailAddress="test@example.com",
+        Destination={
+            "ToAddresses": ["test_to@example.com"],
+            "CcAddresses": ["test_cc@example.com"],
+            "BccAddresses": ["test_bcc@example.com"],
+        },
+        Content={
+            "Simple": {
+                "Subject": {"Data": "test subject"},
+                "Body": {"Text": {"Data": "test body"}},
+            },
+        },
+    )
+
+    # Execute
+    with pytest.raises(ClientError) as e:
+        conn.send_email(**kwargs)
+        assert e
```

Comment:
```suggestion
    with pytest.raises(ClientError) as e:
        conn.send_email(**kwargs)
    assert e.response.value["Error"]["Code"] == "MessageRejected"
```
This asserts that the correct error is raised. Note the indentation - after `conn.send_email` fails, nothing else is executed. So asserting that the error is correct should happen outside this context.